src/mutations/ScheduledEventMutations.ts

```diff
@@ -0,0 +1,41 @@
+import { logger } from '@esss-swap/duo-logger';
+
+import {
+  ScheduledEventDataSource,
+  ScheduledEventDataSourceError,
+  ScheduledEventDataSourceErrorTypes,
+} from '../datasources/ScheduledEventDataSource';
+import { ScheduledEvent } from '../models/ScheduledEvent';
+import { rejection, Rejection } from '../rejection';
+import { NewScheduledEventInput } from '../resolvers/mutations/CreateScheduledEventMutation';
+
+export default class ScheduledEventMutations {
+  constructor(private scheduledEventDataSource: ScheduledEventDataSource) {}
+
+  // TODO: validate input fields
```

Comment:
Maybe validate and authorize as well in the future.

Reply:
Yep, no auth for now